doc/content/modules/user-manual/pages/release-notes/2025.8.0.adoc

```diff
@@ -158,6 +158,9 @@ package p1 {
 
 - In the _Explorer_ view, fix an issue where the context menu of a tree item was loaded twice.
 
```

Comment:
The cooldown branch allows to merge commits for next release (2025.10.0) before the release of the current release (2025.8.0).
It means this contribution will be part of 2025.10.0. So you should add this into 2025.10.0.adoc.
A dedicated file has been created for that as part of the first commit of the cooldown branch.